src/controllers/users/getContentsAndUsers.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response, NextFunction } from 'express';
+import { User, Content } from 'db-models-nc';
+import { USER_ROLES } from '../../helpers/constants';
+
+export default async (_req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
+    const [contents, users] = await Promise.all([Content.findAll(), User.findAll({
+      attributes: ['id', 'name', 'email', 'image'],
+      where: { userRoleId: USER_ROLES.COMEDIAN },
+    })]);
```

Comment:
I would prefer if you order the users and contents descendingly to show new ones first as they recently added
```suggestion
    const [contents, users] = await Promise.all([Content.findAll({
    order: [['id', 'DESC']]}), User.findAll({
      attributes: ['id', 'name', 'email', 'image'],
      where: { userRoleId: USER_ROLES.COMEDIAN },
       order: [['id', 'DESC']],
    })]);
```